documentatie/flowcharts/data-entry-forms.md

```diff
@@ -122,27 +120,24 @@ flowchart TD
     go-to-prev-page([go to previous page])
     abort-input([abort input])
```

Comment:
Do we need `abort input` in this flowchart? Not really related to clicking volgende? Right now it feels as if aborting input is one of the options presented in the dialogs, which is not the case

Reply:
I agree that doesn't make sense, I've removed it.